types/echarts/index.d.ts

```diff
@@ -9,13 +9,11 @@ declare namespace echarts {
         renderer?: string
     }): ECharts;
 
-    const graphic: {
+    interface graphic {
```

Comment:
Please use CamelCase for interface.

Like demonstrated here. http://definitelytyped.org/guides/best-practices.html

Reply:
graphic is a value, not a type. You still need to declare a value like `const graphic: Graphic`